locales/fr-FR.json

```diff
@@ -531,7 +531,7 @@
       },
       "ordinal_values": [
         "1er",
-        "2e",
+        "2nd",
```

Comment:
revert

Reply:
oops, I thought I caught them all. was changing them manually from transifex's output.

Reply:
done. also caught another "2nd" in a phrase, changed that too